### PR TITLE
Added Some content related to facts about pizza

### PR DESCRIPTION
**Pull request by kumar007ambi**
## Description
I have added some content related to facts about pizza.

## What type of PR is this? (check all applicable)

- [ ☑] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

#36 Add pizza fact or trivia
## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by kumar007ambi** [comment]
@CBID2 please review this PR😀.

File: pizza-facts-&-trivia.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ We've categorized the facts into different sections to help explore the diverse
...
 13. Cheese burst pizzas are a cheese lover's dream, and they gained popularity for their gooey, cheesy center. But did you know that the idea for cheese burst pizza was actually inspired by the 'stuffed crust' innovation? In the early 1990s, Pizza Hut introduced stuffed crust pizza, and it was so successful that it led to the creation of the cheese burst pizza, where the entire crust is filled with a delicious, oozing blend of cheeses. So, next time you enjoy a cheese burst pizza, you can thank the stuffed crust for paving the way to this extra cheesy delight
+14. An International Pizza Expo takes place each year in Las Vegas, Nevada. It’s the world’s largest gathering of members from the pizza industry. Unfortunately, it’s not open to the public. You have to be in the pizza business to attend.
+15. Domino’s Pizza Is the Most Popular Pizza Chain in the world as of 2017. The world’s biggest pizza chain in terms of global sales is Domino’s Pizza. The company has more than 17,000 pizza stores in 90 countries, and this number continues to grow.
```
Comment (CBID2) [inline pizza-facts-&-trivia.md:25]:
```suggestion
15. Domino’s Pizza is the Most Popular Pizza Chain in the world as of 2017. The company has more than 17,000 pizza stores in 90 countries, and this number continues to grow.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
You're off to a good start @kumar007ambi. There are some things that need to be improved. Also, there some commits in there from your PR about Indian pizza in this branch. I highly recommend removing them.

**Comment by adiati98** [comment]
Hi @kumar007ambi,

We've merged a PR before yours. So please [resolve the conflicts](https://github.com/open-sauced/pizza-verse#keeping-branch-up-to-date-and-resolving-merge-conflicts) and fix the suggestion from @CBID2 before we can merge your PR. :)

Also be aware that we're now using unordered list with dash (-) instead of ordered list (number).
Thank you :)

**Comment by kumar007ambi** [comment]
@CBID2 and @adiati98 I have edited my content according to suggestion.

File: pizza-facts-&-trivia.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ We've categorized the facts into different sections to help explore the diverse
...
 - Cheese burst pizzas are a cheese lover's dream, and they gained popularity for their gooey, cheesy center. But did you know that the idea for cheese burst pizza was actually inspired by the 'stuffed crust' innovation? In the early 1990s, Pizza Hut introduced stuffed crust pizza, and it was so successful that it led to the creation of the cheese burst pizza, where the entire crust is filled with a delicious, oozing blend of cheeses. So, next time you enjoy a cheese burst pizza, you can thank the stuffed crust for paving the way to this extra cheesy delight
+- An International Pizza Expo takes place each year in Las Vegas, Nevada. It’s the world’s largest gathering of members from the pizza industry. Unfortunately, it’s not open to the public. You have to be in the pizza business to attend.
+- Domino’s Pizza is the Most Popular Pizza Chain in the world as of 2017. The company has more than 17,000 pizza stores in 90 countries, and this number continues to grow.
```
Comment (CBID2) [inline pizza-facts-&-trivia.md:24]:
```suggestion
- As of 2017, Domino’s Pizza is the most popular pizza chain in the world. The company has more than 17,000 pizza stores in 90 countries, and this number continues to grow.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
You're close @kumar007ambi! :)

**Comment by kumar007ambi** [comment]
@CBID2 I have edited my content according to suggestion.

**Review by CBID2 — APPROVED** [review]
Looks great

**Pull request merged**